Framework/Algorithms/inc/MantidAlgorithms/Stitch1DMany.h

```diff
@@ -94,13 +93,13 @@ class DLLExport Stitch1DMany : public API::Algorithm {
   std::vector<double> m_endOverlaps;
   std::vector<double> m_params;
   std::vector<double> m_scaleFactors;
+  std::vector<double> m_manualScaleFactors;
```

Comment:
Hmm, the purpose of the `doStitch1D()` and `doStitch1DMany()` was to provide functions for running the `Stitch1D` and `Stitch1DMany` algorithms respectively. The parameters for those functions were named after the same properties they supply. It's not exactly easy to explain, but I don't think its a good idea to change it as it makes less sense semantically to me.